script/run-fuzzer

```diff
@@ -29,7 +29,7 @@ run_fuzzer() {
   # then be loaded on subsequent fuzzing runs
   mkdir -p corpus
 
-  "../../${lang}_fuzzer_${mode}" "-dict=../../${lang}.dict" "-artifact_prefix=${lang}_${mode}_" -max_len=2048 ${mode_config[$mode]} "./corpus" "$@"
```

Comment:
Can you comment on why these changes were necessary?

Reply:
I guess they are not strictly necessary for fuzzing. Essentially I was figuring out how to build and run the fuzzers in order to write the Dockerfile and Mayhemfiles, so I was referring to the `run-fuzzer` script they provided. The script would not run correctly without this modification, which is how it made it in. I am pretty sure the script failing is a matter of it being out of date. Thus I'm not sure if my modification should be committed.

Reply:
Looking more at this, I don't believe this change is necessary - You need to provide both a language and a "mode" (see here: https://github.com/mayhemheroes/tree-sitter/blob/80a925727bd836f11a32bb1e75196f41bffc37e2/script/run-fuzzer#L13).
Please revert this change as it doesn't seem to be blocking anything and the script doesn't appear to be used in your Dockerfile.

Reply:
When I ran the script, I did provide both a language and a mode. You can see that my change to the line was not deleting it, but rather removing the `_${mode}` suffix from the fuzzer executable. That is because the `build-fuzzers` script I invoke does not output fuzzer executables with that suffix.

But you are right, since I don't use it in the Dockerfile, I will remove it.